back/app/services/project_copy_service.rb

```diff
@@ -73,6 +76,54 @@ def export(
 
   private
 
+  def yml_content_builder_layouts(shift_timestamps: 0)
+    layout_images_mapping = {}
+
+    layouts = ContentBuilder::Layout.where(content_buildable_id: @project.id).map do |layout|
+      craftjs = layout.craftjs_jsonmultiloc
+
+      craftjs.each_key do |locale|
+        craftjs[locale].each_key do |node|
```

Comment:
Little thing, but you could use `each_value do |node|` and use `node` instead of `craftjs[locale][node]`.

Reply:
Really nice 'little thing'! This cleans up the code nicely. 😄